catroid/src/test/java/org/catrobat/catroid/test/content/bricks/UpdateVariableInFormulaBrickTest.java

```diff
@@ -114,6 +114,8 @@ public void testRenamingVariable() {
 			formulaBrick.setFormulaWithBrickField(k, newFormula);
 		});
 
+		ProjectManager.getInstance().getCurrentProject().addUserVariable(userVariable);
```

Comment:
Is it really necessary to add the userVariable in each testcase? I would prefer to add it in the setUp() method by simply calling project.addUserList(userList) and project.addUserVariable(userVariable) once. Same applies for all the other Tests.